Instructions/movement_instruction.py

```diff
@@ -0,0 +1,32 @@
+from collections import deque
+
+# Ours
+from ..Tasks.movement_task import MovementTask
+from instruction_base import InstructionBase
+from ..Utilities import constants as c
+
+class MovementInstruction(InstructionBase):
+
+    def __init__(self, data):
+        super(MovementInstruction, self).__init__()
+        # The data here will eventually be the raw binary data that
+        # has traveled over the network. The decoding may be more
+        # involved than what is shown here.
+        (self.x, self.y, self.z) = data
```

Comment:
I believe the parentheses are not necessary here
```suggestion
        self.x, self.y, self.z = data
```

Reply:
The code runs fine without them, so they have been removed.